event_webhook_types.go

```diff
@@ -246,6 +247,8 @@ type IssueCommentEvent struct {
 		HumanTimeEstimate   string        `json:"human_time_estimate"`
 		AssigneeIDs         []int         `json:"assignee_ids"`
 		AssigneeID          int           `json:"assignee_id"`
+		Position            int           `json:"position"`
+		BranchName          string        `json:"branch_name"`
```

Comment:
Can you order them according to the docs? So in between `AuthorID` and `Description`? Thanks!

Reply:
Fixed order.